runtime/executor/src/main/java/org/apache/nemo/runtime/executor/data/DataUtil.java

```diff
@@ -304,6 +305,27 @@ public long getNumEncodedBytes() {
       }
       return numEncodedBytes;
     }
+
+    @Override
+    public long getCurrNumSerializedBytes() {
+      if (serializedCountingStream == null) {
+        return numSerializedBytes;
+      }
+      return numSerializedBytes + serializedCountingStream.getCount();
+    }
+
+    @Override
+    public long getCurrNumEncodedBytes() {
```

Comment:
What is the difference between getNumEncodedBytes and getCurrNumEncodedBytes?

Reply:
It has same definition and reason with getCurrNumSerializedBytes